experimental/packages/exporter-trace-otlp-grpc/README.md

```diff
@@ -3,7 +3,7 @@
 [![NPM Published Version][npm-img]][npm-url]
 [![Apache License][license-image]][license-image]
 
-This module provides exporter for web and node to be used with [opentelemetry-collector][opentelemetry-collector-url].
+This module provides exporter for node to be used with [opentelemetry-collector][opentelemetry-collector-url].
```

Comment:
Not related to this fix, but I think the wording should be "OTLP receiver" instead of "opentelemetry-collector", since the exporter can send to any OTLP compatible receiver.

Reply:
Probably in other packages as well. Anyway, just sharing a thought. I can open a separate PR after this one is merged.

Reply:
Yes, that makes sense. :slightly_smiling_face: 
Since this PR is already approved I can also do it in a follow-up if you like.